loop/simplepush.js

```diff
@@ -26,23 +26,17 @@ SimplePush.prototype = {
     var self = this;
 
     urls.forEach(function(simplePushUrl) {
-      if (self.statsdClient !== undefined) {
-        self.statsdClient.count("loop.simplepush.call", 1);
-        self.statsdClient.count("loop.simplepush.call." + reason, 1);
-      }
       request.put({
         url: simplePushUrl,
         form: { version: version }
       }, function(err) {
+        var status = 'success';
+        if (err) {
+          self.logError(err);
```

Comment:
Nit: not introduced by this patch by you should probably set the third argument of `forEach` to `instead` instead of relying on `self` in this whole loop.